lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -6,7 +6,9 @@
   raise "Could not load the braintree gem.  Use `gem install braintree` to install it."
 end
 
-raise "Need braintree gem 2.x.y. Run `gem install braintree --version '~>2.0'` to get the correct version." unless Braintree::Version::Major == 2
+unless Braintree::Version::Major == 2 && Braintree::Version::Minor >= 4
```

Comment:
We could probably require a higher version than `2.4.0` if we wanted. I noticed at least `2.50.0` is [required by the ActiveMerchant Gemfile](https://github.com/activemerchant/active_merchant/blob/a405e03/Gemfile#L8).